dbt_project.yml

```diff
@@ -771,6 +783,111 @@ models:
         +schema: zeroex_arbitrum
 
 
+    ampleforth:
+      +schema: ampleforth
+      ethereum:
+        +schema: ampleforth_ethereum
+
+    apecoin:
+      +schema: apecoin
+      ethereum:
+        +schema: apecoin_ethereum
+
+    forta_network:
+      +schema: forta_network
+      ethereum:
+        +schema: forta_network_ethereum
+
+    gas_dao:
+      +schema: gas_dao
+      ethereum:
+        +schema: gas_dao_ethereum
+
+    ribbon:
+      +schema: ribbon
+      ethereum:
+        +schema: ribbon_ethereum
+
+    galxe:
+      +schema: galxe
+      ethereum:
+        +schema: galxe_ethereum
+
+    gearbox:
+      +schema: gearbox
+      ethereum:
+        +schema: gearbox_ethereum
+
+    snowswap:
+      +schema: snowswap
+      ethereum:
+        +schema: snowswap_ethereum
+
+    arbitrum:
```

Comment:
i think for blockchain-level airdrops, we can stick to just one level in the hierarchy, rather than repeat it twice -- wdyt?

Reply:
if changed here, of course all the models in the directory it relates to need changed as well

Reply:
What do you mean exactly? What is repeated twice and where?

Reply:
arbitrum directory within the arbitrum directory -- i.e. `models/arbitrum/arbitrum/`. i think we can just stop at `models/arbitrum/` and do it all within there and simplify naming standards on files names, directories, refs(), etc

Reply:
Yeah I get how than can be confusing but imo its the most futureproof method, I bet we will eventually see a blockchain airdrop its native token on a chain other than its own/multiple chains!

Also, Optimism has `op_optimism` for their token which doesn't follow the dune structure of `protocol_chain`, I was thinking that it should be renamed but it could be renamed to either `optimism_optimism` or `optimism`, imo the most futureproof is the most detailed (`optimism_optimism`), but maybe @MSilb7 & @chuxinh have some thoughts here?